scenariolib/visit.go

```diff
@@ -408,9 +408,10 @@ func (v *Visit) FindDocumentRankByTitle(toFind string) int {
 // WaitBetweenActions Wait a random or constant number of seconds between user actions
 func WaitBetweenActions(timeToWait int, isConstant bool) {
 	if isConstant {
-		time.Sleep(time.Duration(timeToWait) * time.Second)
+		time.Sleep((time.Duration(timeToWait*1000) + 500) * time.Second)
```

Comment:
Sorry you are right about the `*time.Second`, I re-changed it to `*time.Millisecond`

But no the input of time to wait is still in seconds as far as the configuration is concerned, that's why I multiply it by 1000 to then multiply it by `time.Millisecond`.

So effectively in the config you will input:
TimeBetweenActions: 3`
So it will become `3*1000*time.Millisecond` so 3000 Milliseconds => 3 seconds.

I changed it so that it will be minimum 500 Milliseconds instead of always +500 Milliseconds.